### PR TITLE
[Merged by Bors] - port: `Nat.sqrt` extension for `norm_num`

### PR DESCRIPTION
**Pull request by kmill**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by kmill** [comment]
@digama0 Here's a port of your `Nat.sqrt` extension. Should I be including any porting header for this meta file since it came from the theory folder?

File: test/norm_num_ext.lean [hunk 1]
```diff
@@ -8,30 +8,34 @@ Authors: Mario Carneiro
 import Mathlib.Data.Nat.Fib
 import Mathlib.Data.Nat.Prime
 import Mathlib.Data.Nat.Factors
---import Mathlib.Data.Nat.SqrtNormNum
 --import Mathlib.Analysis.SpecialFunctions.Pow
 --import Mathlib.NumberTheory.LegendreSymbol.NormNum
 import Mathlib.Tactic.NormNum.GCD
 import Mathlib.Tactic.NormNum.IsCoprime
+import Mathlib.Tactic.NormNum.NatSqrt
+import Std.Tactic.ShowTerm
```
Comment (digama0) [inline test/norm_num_ext.lean:16]:
```suggestion
```

**Comment by semorrison** [comment]
I think no need for a porting header.

**Pull request closed without merging**